cpp/models/ode_seir/parameters.h

```diff
@@ -108,6 +108,35 @@ class Parameters : public ParametersBase
     {
     }
 
+    /**
+     * @brief checks whether all Parameters satisfy their corresponding constraints and applies them, if they do not
+     * @return Returns 1 if one ore more constraint was corrected, 0 otherwise.  
+     */
+    int apply_constraints()
+    {
+        int corrected = 0;
+        if (this->get<TimeExposed>() <= 0.0) {
+            log_warning("Constraint check: Parameter TimeExposed changed from {:.4f} to {:.4f}",
```

Comment:
Above your constraint for times is <1.0 and not <= 0.0

Reply:
I thought about it again and did some tests.
Small values for the time intervalls (even 1e-2) leads to an insane increase of runtime and often results in nan errors. So asking for <1 makes sense in my opinion to provide a stable simulation with good run times.
Therefore i adjusted the constraints in the seir part to make it consistent.

Reply:
Using the secir_ageres example with T = 50.
Because of the more complex restrictrions for the SI and INC, i have just set
TimeInfectedSymptoms, TimeInfectedSevere and TimeInfectedCritical to one MinVal.

MinVal   Num Time points
1	      59
0.1 	      158
0.01	      1354
1e-5       1323782
Apply the minVal to only one of these times leads to similar results, so they are all equal problematic.

In my opinion, one of the main target groups of this function are users who are insecure in handling the parameters and therefore I would prefer the choice with greater than 1 and thus more stable runtime.